### ISSUE TITLE
CamelCase vs snake_case -- Naming Convenion -- Renaming unhcRverse to unhcrverse

### ISSUE THREAD

**Issue by dickoa** [unhcRverse/unhcrverse#1]
Hi everyone,

Thanks for setting this repository for our discussions.

I wanted to propose that we rename the organization and meta-package to `unhcrverse` instead of `unhcRverse`.

The first reason we want to rename is that it follows more modern idioms as seen in e.g tidyverse, ropensci, r-lib, etc.
It's much easier to type, no risk of trying to find which letter to capitalize, etc. Many other points are summarized in this blog post:

https://www.njtierney.com/post/2018/06/20/naming-things/

More generally, it's linked to CamelCase vs snake_case, and adopt a proper style for the `unhcrverse`.

What do you think?

**Comment by Edouard-Legoupil** [comment]
Indeed an important point - maybe to be further documented here: https://unhcrverse.github.io/unhcrverse/articles/codingstyle.html = 

Note that the package itself, as it is now is not using any capital letter - cf https://github.com/unhcrverse/unhcrverse/blob/main/DESCRIPTION#L2 

One challenge is that snake_case are actually not allowed for package name... - https://stackoverflow.com/questions/24201568/what-constitutes-a-good-package-name-according-to-cran

**Comment by vidonne** [comment]
I don't know if it's case for you but the change only appears on the org homepage for me. The actual URL of the org as well as the package still have the capital letter. We can also see it in the call to load the package.

**Comment by dickoa** [comment]
@Edouard-Legoupil I agree that we should not use snake case for naming package. Let's work on the style, but I'm pushing for a similar style the tidyverse team is using (also r-lib, etc). 
Just aiming for CRAN compatibility, you can have a package with dots i.e {data.table}, I don't think it's a good idea (confuse with S3 methods). With a proper coding style, we'll have more guidance on it. 
For the organization, it's easier to quickly change it from the settings, same for the repository of the package right?

**Comment by Edouard-Legoupil** [comment]
Done! all renamed! - https://unhcrverse.github.io/unhcrverse/articles/codingstyle.html would be the place to provide more details on all coding convention! happy to be an humble follower on this!

**Comment by Edouard-Legoupil** [comment]
Hello - 

Based on recommendations 

 * Package name - all small cap - with eventually dot separator -  (as snake_case are not allowed for packages)
 * variables within package - snake_case all over
 * CamelCase only to be used within R6 classes
